examples/ads.amp.html

```diff
@@ -401,13 +401,25 @@ <h2 id="contentad">Content.ad Banner 320x50</h2>
     <div fallback></div>
   </amp-ad>
 
-  <h2 id="criteo">Criteo (0x0 fake ad)</h2>
+  <h2 id="criteo">Criteo Passback</h2>
   <amp-ad width="300" height="250"
       type="criteo"
+      data-tagtype="passback"
```

Comment:
It will be better if you can make this particular ad slot always load a fake ad. Not only it will be a more confident example for publishers to follow, but also we would be able to catch any regression bug during our releases. 

If it's a feature currently missing in your ad server, I'm fine to leave it as is. It's just we have much less chance to catch a bug. If you're fine with it, please have one line to call it out between the ad slot and its title:

```
Due to ad targeting, the slot might not load ad.
```